### PR TITLE
CIV-12784 Dashboard Notification - HwF - When Case worker selects "Invalid HwF Number"

### PR DESCRIPTION
**Pull request by manish14836**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-12784

### Change description ###

image

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/main/resources/db/migration/V2024_03_01_1222__CIV-12657-claim-issue-hwf-scenario.sql [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+/**
+ * Add task list items
+ */
+INSERT INTO dbs.task_item_template (task_name_en, category_en, task_name_cy, category_cy, template_name,
+                                    scenario_name, task_status_sequence, role,task_order)
+values ('<a href={VIEW_CLAIM_URL}  rel="noopener noreferrer" class="govuk-link">View the claim</a>', 'The claim',
+        '<a href={VIEW_CLAIM_URL}  rel="noopener noreferrer" class="govuk-link">View the claim</a>',
+        'The claim', 'Claim.View', 'Scenario.AAA7.ClaimIssue.HWF.Requested', '{3, 3}', 'CLAIMANT', 1),
+       ('<a href={VIEW_INFO_ABOUT_CLAIMANT_URL}  rel="noopener noreferrer" class="govuk-link">View information about the claimant</a>', 'The claim',
```
Comment (paul-pearson) [inline src/main/resources/db/migration/V2024_03_01_1222__CIV-12657-claim-issue-hwf-scenario.sql:26]: I think the placeholder may need to be `VIEW_INFO_ABOUT_CLAIMANT` to match CUI dashboardInterpolationService.ts `setDashboardValues()` function.

File: src/main/resources/db/migration/V2024_03_01_1222__CIV-12657-claim-issue-hwf-scenario.sql [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        '<a href={VIEW_INFO_ABOUT_CLAIMANT_URL}  rel="noopener noreferrer" class="govuk-link">View information about the claimant</a>',
+        'The claim', 'Claim.Claimant.Info', 'Scenario.AAA7.ClaimIssue.HWF.Requested', '{3, 3}', 'CLAIMANT', 2),
+       ('<a>View the response to the claim</a>', 'The response','<a>View the response to the claim</a>',
+        'The response', 'Response.View', 'Scenario.AAA7.ClaimIssue.HWF.Requested', '{1, 1}', 'CLAIMANT', 3),
+       ('<a href={VIEW_INFO_ABOUT_DEFENDANT_URL}  rel="noopener noreferrer" class="govuk-link">View information about the defendant</a>', 'The response',
```
Comment (paul-pearson) [inline src/main/resources/db/migration/V2024_03_01_1222__CIV-12657-claim-issue-hwf-scenario.sql:31]: I think the placeholder may need to be `VIEW_INFO_ABOUT_DEFENDANT` to match CUI dashboardInterpolationService.ts `setDashboardValues()` function.

File: src/main/resources/db/migration/V2024_03_01_1222__CIV-12657-claim-issue-hwf-scenario.sql [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        'Hearings', 'Hearing.View', 'Scenario.AAA7.ClaimIssue.HWF.Requested', '{1, 1}', 'CLAIMANT', 5),
+       ('<a>Upload hearing documents</a>', 'Hearings' ,'<a>Upload hearing documents</a>',
+        'Hearings', 'Hearing.Document.Upload', 'Scenario.AAA7.ClaimIssue.HWF.Requested', '{1, 1}', 'CLAIMANT', 6),
+       ('<a>Add the trial arrangements</a>', 'Hearings' ,'<a>Add the trial arrangements</a>',
+        'Hearings', 'Hearing.Arrangements.Add', 'Scenario.AAA7.ClaimIssue.HWF.Requested', '{1, 1}', 'CLAIMANT', 7),
+       ('<a>Pay the hearing fee</a>', 'Hearings' ,'<a>Pay the hearing fee</a>',
+        'Hearings', 'Hearing.Fee.Pay', 'Scenario.AAA7.ClaimIssue.HWF.Requested', '{1, 1}', 'CLAIMANT', 8),
+       ('<a>View the bundle</a>', 'Hearings' ,'<a>View the bundle</a>',
+        'Hearings', 'Hearing.Bundle.View', 'Scenario.AAA7.ClaimIssue.HWF.Requested', '{1, 1}', 'CLAIMANT', 9),
+       ('<a href={VIEW_ORDERS_AND_NOTICES_URL}  rel="noopener noreferrer" class="govuk-link">View orders and notices</a>', 'Orders and notices from the court' ,
```
Comment (paul-pearson) [inline src/main/resources/db/migration/V2024_03_01_1222__CIV-12657-claim-issue-hwf-scenario.sql:44]: As above - `VIEW_ORDERS_AND_NOTICES`?

**Review by paul-pearson — CHANGES_REQUESTED** [review]

File: src/main/resources/db/migration/V2024_03_01_1222__CIV-12657-claim-issue-hwf-scenario.sql (hunk 1 above)
Comment (manish14836) [reply src/main/resources/db/migration/V2024_03_01_1222__CIV-12657-claim-issue-hwf-scenario.sql:26]: This change is from another PR https://github.com/hmcts/civil-service/pull/4136

File: src/main/resources/db/migration/V2024_03_01_1222__CIV-12657-claim-issue-hwf-scenario.sql (hunk 2 above)
Comment (manish14836) [reply src/main/resources/db/migration/V2024_03_01_1222__CIV-12657-claim-issue-hwf-scenario.sql:31]: This change is from another PR https://github.com/hmcts/civil-service/pull/4136

File: src/main/resources/db/migration/V2024_03_01_1222__CIV-12657-claim-issue-hwf-scenario.sql (hunk 3 above)
Comment (manish14836) [reply src/main/resources/db/migration/V2024_03_01_1222__CIV-12657-claim-issue-hwf-scenario.sql:44]: This change is from another PR https://github.com/hmcts/civil-service/pull/4136

File: src/main/resources/db/migration/V2024_03_01_1222__CIV-12657-claim-issue-hwf-scenario.sql (hunk 1 above)
Comment (manish14836) [reply src/main/resources/db/migration/V2024_03_01_1222__CIV-12657-claim-issue-hwf-scenario.sql]: Updated in another PR

**Review by neeta-hmcts — APPROVED** [review]

**Review by kalachandrasekar1 — APPROVED** [review]

**Review by paul-pearson — APPROVED** [review]

**Review by dharmendrak — APPROVED** [review]

**Pull request merged**